src/peft/tuners/lora/tp_layer.py

```diff
@@ -0,0 +1,160 @@
+from typing import Any
+
+import torch
+import torch.nn as nn
+import torch.nn.init as init
+
+from .layer import LoraLayer
+
+
+class LoraParalleLayer(LoraLayer):
+    def __init__(self, base_layer, backend, is_paralle_a: bool = False):
+        LoraLayer.__init__(self, base_layer=base_layer)
```

Comment:
This should also work, right?

```suggestion
        super().__init__(base_layer=base_layer)
```

Reply:
> This should also work, right?

Yes, but now `LoraParalleLayer` will be removed.